packages/webhooks/test/commands/webhooks/deliveries/index.test.ts

```diff
@@ -48,10 +48,12 @@ describe('webhooks:deliveries', () => {
     .command(['webhooks:deliveries', '--app', 'example-app'])
     .it('lists webhooks deliveries for app webhooks', ctx => {
       expect(ctx.stderr).to.equal('')
-      expect(ctx.stdout.trim()).to.equal(
-        `Delivery ID                          Created              Status   Include   Level  Attempts Code Error  Next Attempt         
-99999999-9999-9999-9999-999999999999 2017-08-17T20:22:37Z retrying api:build notify 4        401  Foobar 2017-08-17T20:22:39Z 
-66666666-6666-6666-6666-666666666666 2017-08-17T20:22:38Z pending  api:build notify 4`)
+      expect(ctx.stdout).to.contain('Delivery ID')
+      expect(ctx.stdout).to.contain('Created')
+      expect(ctx.stdout).to.contain('Status   Include   Level  Attempts Code Error  Next Attempt')
+      expect(ctx.stdout).to.contain(' 99999999-9999-9999-9999-999999999999 2017-08-17T20:22:37Z retrying api:build notify 4')
+      expect(ctx.stdout).to.contain('401  Foobar 2017-08-17T20:22:39Z')
+      expect(ctx.stdout).to.contain(' 66666666-6666-6666-6666-666666666666 2017-08-17T20:22:38Z pending  api:build notify 4')
```

Comment:
Like above, Split these lines up because the tests don't handle \n correctly with the oclif upgrade. Changed ` to ' because linting complained about it (even though I did not change eslint version)